sharedb.planx.uk/package.json

```diff
@@ -4,9 +4,11 @@
   "private": true,
   "dependencies": {
     "@teamwork/websocket-json-stream": "^2.0.0",
+    "dompurify": "^3.0.6",
+    "jsdom": "^23.0.0",
     "jsonwebtoken": "^8.5.1",
     "pg": "^8.11.3",
-    "sharedb": "^3.3.1",
+    "sharedb": "^4.1.1",
```

Comment:
The only breaking changes in v4 is dropped support for Node 14.

Docs: https://github.com/share/sharedb/releases/tag/v4.0.0